docs/components/sidecar.md

```diff
@@ -79,6 +79,9 @@ To use this, the Prometheus compaction needs to be disabled. This can be done by
 - `--storage.tsdb.min-block-duration=2h`
 - `--storage.tsdb.max-block-duration=2h`
 
+A known issue is that Prometheus and Thanos use different compaction schedules, so Compact may
+get confused when trying to further compact or downsample compacted blocks from Prometheus.
```

Comment:
```suggestion
get confused when trying to further compact or downsample compacted blocks from Prometheus.
This is similar to why enabling local compaction with upload is tricky.

This can be overcome by enabling experimental vertical compaction.
```